practica2023/src/main/java/com/endava/practica2023/Domain/Eventss.java

```diff
@@ -0,0 +1,98 @@
+package com.endava.practica2023.Domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+import java.util.Set;
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+@Builder
+@Table(name = "eventss")
+public class Eventss {
+    @Id
+    @Column(name = "event_id")
+    private Integer eventId;
+
+    @Column(name = "event_description")
+    private String eventDescription;
+
+    @Column(name = "event_name")
+    private String eventName;
+
+    @Column(name = "start_date")
+    private LocalDateTime startDate;
+
+    @Column(name = "end_date")
+    private LocalDateTime endDate;
+
+    @ManyToOne
+    @JoinColumn(name = "venue_id")
+    private Venue venue;
+
+    @ManyToOne
+    @JoinColumn(name = "event_type_id")
+    private EventType eventType;
+
+    @OneToMany(mappedBy = "eventss")
+    @JsonIgnore
+    private Set<TicketCategory> ticketCategory;
+
+    public Integer getEventId() {
```

Comment:
Why didn't you use lombok to have as well getter and setters and not create them here?